src/index.js

```diff
@@ -6,6 +6,17 @@ import { PublicUrl, BaseStyles } from '@aragon/ui'
 import GlobalErrorHandler from './GlobalErrorHandler'
 import App from './App'
 
+const APP_VERSION = '0.5.4'
+const APP_VERSION_KEY = 'APP_VERSION'
+
+if (window && window.localStorage) {
+  const lastAppVersion = window.localStorage.getItem(APP_VERSION_KEY)
+  if (lastAppVersion !== APP_VERSION) {
```

Comment:
In the body of the PR you mention only purging on minor versions, should we check that here? or are you thinking that we would only change the `APP_VERSION` value on minor versions?

Reply:
Good point, yeah let's do it based on minor versions for now. I'm hoping this bit of code doesn't last particularly long :).